installation-english.md

```diff
@@ -0,0 +1,72 @@
+# Swipe Installation 
+
+Install the Swipe console utility: 
+````javascript
```

Comment:
Maybe change to shell/bash instead of JavaScript?

Reply:
That I can do, I am still learning all the languages Markdown will take, and learning to read what I see in github with no traditional background in computers.